### DISCUSSION TITLE
If there is any plan for object storage like Amazon S3 to store QSL card images?

### DISCUSSION THREAD

**Discussion by samhjn** [magicbug/Cloudlog#1364] [Ideas / Feature Requests]
Hi, thanks for developing cloudlog, which allows me to bring up a web-based ham radio log system rapidly.

I have noticed that cloudlog uses local file system to storage QSL card images. It works and meets most situations fine. However, object storage serivce like Amazon S3 or Alibaba Cloud OSS is much cheaper than block storage on cloud platforms, and may allows cloudlog to decouple web server deployment with QSL card image storage.

I don't know whether there is any heavy users for this function, or how big the storage size will grow, but obviously it will be add up the complexity of the file management modules. So it is hard for me to say whether these features are make sense for cloudlog.

So I want to know if there is any plan for object storage? If object storage support is welcome, I may take some time to support it.

73, de BH5HSU

**Comment by magicbug** [comment]
Its something I've considered but other things have always come first feature-wise, but certainly, if someone would like to explore adding this functionality then it will get full support by myself.

I know of a few people who do use it quite heavily, I'm in the process of scanning all my cards to assign against QSOs and its using a good amount of space :)

Peter.
